notebooks/electrical_characterization_sky130_epfl.ipynb

```diff
@@ -0,0 +1,1204 @@
+{
+ "cells": [
```

Comment:
can we add a new cell with a colab badge:
```
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "view-in-github",
+    "colab_type": "text"
+   },
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/proppy/skywater-pdk-sky130-raw-data/blob/develop/notebooks/electrical_characterization_sky130_epfl.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+   ]
+  },
```
and another one that clone the repo, similar to what we have for the other notebooks:
https://github.com/google/skywater-pdk-sky130-raw-data/blob/main/notebooks/sky130_plot_gm.ipynb
https://github.com/google/skywater-pdk-sky130-raw-data/blob/main/notebooks/sky130_plot_current.ipynb

also maybe rename it to have s similar naming scheme as the other one?

@bmurmann should we look into consolidating with your existing notebook and allow it to work across multiple dataset?

Reply:
My notebook is very simple, best to build on this new one instead of consolidating/merging.

Reply:
Got it. Notebooks were renamed in [04d30b1](https://github.com/google/skywater-pdk-sky130-raw-data/pull/23/commits/04d30b1393ecb968d54b4ff09c54503879b9db61).
Also, README was updated correspondingly in [f6bc6ab](https://github.com/google/skywater-pdk-sky130-raw-data/pull/23/commits/f6bc6ab632dfe5e4db7e2f77cf0c839ce1d0708d)